### DISCUSSION TITLE
new Evaluation group (interval)

### DISCUSSION THREAD

**Discussion by AkakievKD** [grafana/grafana#72769] [Alerting]
how to create a new Evaluation group (interval) i want to have more one Evaluation group (interval)

![Screenshot_8](https://github.com/grafana/grafana/assets/82488489/1ebd32c1-bb02-46f4-9e07-2ec82fee24e3)

![Screenshot_11](https://github.com/grafana/grafana/assets/82488489/154ec0e7-d563-4396-95af-cbeb59e5837f)

**Comment by AkakievKD** [comment]
i want to create a new Evaluation group (interval) just like the folders on the left

**Reply by konrad147** [reply]
You can do that - start typing in the input, and the new option to create a group should pop up.
We've improved this section and the changes will be available in Grafana v10.1
Related PR https://github.com/grafana/grafana/pull/71251

**Comment by AkakievKD** [comment]
Thanks
